integration_tests/test_str_01.py

```diff
@@ -1,3 +1,5 @@
+from lpython import i32
```

Comment:
@Agent-Hellboy Is the above import being used in the test case? If it is unused, I suggest we remove it.

Reply:
Done